README.md

```diff
@@ -1,26 +1,34 @@
-# DZ Community Bot 🤖 
+# DZ Community Bot 🤖
 > A bot for AlgeriaTech.slack.com
 
 The goal of this project is to invite the community to collaborate on building something useful and fun.
-The bot should welcome new members of the team and keep the community active in different ways (news, debates, 
+The bot should welcome new members of the team and keep the community active in different ways (news, debates,
```

Comment:
Please keep the double spaces in EOL [5,14,16,18]. They are intended to go back to a new line :)
I've set the `.editorconfig` to take account of this.